src/angularjs/src/app/places/list/place-list.html

```diff
@@ -10,20 +10,32 @@ <h2 class="sidebar-title">City/Town Search</h2>
         <div class="row">
             <div class="column">
                 <div class="form-group">
-                    <label for="1">Filter by city or town name</label>
-                    <input type="text" class="form-control">
+                    <label for="neighborhood-filter">Filter by city or town name</label>
+                    <ui-select
+                        name="neighborhood-filter"
+                        ng-model="placeList.neighborhoodFilter"
+                        title="Filter by Neighborhood">
+
+                        <ui-select-match allow-clear="true" placeholder="Neighborhood" >
+                            {{$select.selected.label}}
+                        </ui-select-match>
+                        <ui-select-choices repeat="neighborhood.uuid as neighborhood in
```

Comment:
Yeah, if there's going to be a large list here, implementing this name filter as a standard typeahead instead of this ui-select filter might be better.

We need to address the permissions of what neighborhoods are returned from the request used here separately, that will have to wait until we revamp the permissions next sprint, so that doesn't need to be done now.